_pkgdown.yml

```
@@ -21,29 +21,29 @@ navbar:
       href: https://github.com/insightsengineering/teal.reporter
 
 reference:
-  - title: Shiny modules for adding content to reports
+  - title: `shiny` modules for adding content to reports
```

```suggestion
  - title: >
      `shiny` modules for adding content to reports
```

The same change for the titles below as well.

---

Why? :thinking:

---

@vedhav Can you explain this suggested change please?

---

The pkgdown will complain starting the title with a backtick.